lib/src/common/github.dart

```diff
@@ -331,7 +344,11 @@ class GitHub {
     var request = http.Request(method, Uri.parse(url.toString()));
     request.headers.addAll(headers);
     if (body != null) {
-      request.body = body;
+      if (body is String) {
+        request.body = body;
+      } else {
+        request.bodyBytes = body;
```

Comment:
at this point `body` can be any type except a String .. and this assignment may result in a runtime error.